src/AppBundle/Entity/Team.php

```diff
@@ -372,4 +401,13 @@ public function setApplications(TeamApplication $applications): void
     {
         $this->applications = $applications;
     }
+
+    /**
+     * @return bool
+     */
+    public function getAcceptApplicationAndDeadline()
+    {
+        $now = new \DateTime();
+        return (($this->acceptApplication and $now < $this->deadline) or ($this->acceptApplication and $this->deadline == null));
+    }
```

Comment:
&& og || i stedet for and og or

> The and operator does not have the same precedence as &&. This could lead to unexpected behavior, use && instead. 

> The or operator does not have the same precedence as ||. This could lead to unexpected behavior, use || instead.

https://insight.symfony.com/projects/ec066ae0-5658-4365-af0c-4ab7492d2ad7/analyses/2724?status=commit